data/xml/2012.amta.xml

```diff
@@ -0,0 +1,638 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2012.amta">
+  <volume id="keynotes" ingest-date="2021-06-01">
+    <meta>
+      <booktitle>Proceedings of the 10th Conference of the Association for Machine Translation in the Americas: Keynote Presentations</booktitle>
+      <publisher>Association for Machine Translation in the Americas</publisher>
+      <address>San Diego, California, USA</address>
+      <month>October 28-November 1</month>
+      <year>2012</year>
+    </meta>
+    <paper id="1">
+      <title>Domain Adaptation in Machine Translation: Findings from the 2012 <fixed-case>J</fixed-case>ohns <fixed-case>H</fixed-case>opkins Summer Workshop</title>
+      <author><first>Hal</first><last>Daumé III</last></author>
+      <author><first>Marine</first><last>Carpuat</last></author>
+      <author><first>Alex</first><last>Fraser</last></author>
+      <author><first>Chris</first><last>Quirk</last></author>
+      <url hash="3c03f46d">2012.amta-keynotes.1</url>
+    </paper>
+    <paper id="2">
+      <title>Trends in Adoption of <fixed-case>MT</fixed-case> as a Globalization Technology</title>
+      <author><first>Don</first><last>DePalma</last></author>
+      <url hash="22480ba8">2012.amta-keynotes.2</url>
+    </paper>
+    <paper id="3">
+      <title>Language Research at <fixed-case>DARPA</fixed-case>-Machine Translation and Beyond</title>
+      <author><first>Bonnie J.</first><last>Dorr</last></author>
+      <url hash="5cf2f073">2012.amta-keynotes.3</url>
+    </paper>
+    <paper id="4">
+      <title><fixed-case>D</fixed-case>uolingo: Learn a Language for Free while Helping to Translate the Web</title>
+      <author><first>Luis von</first><last>Ahn</last></author>
+      <url hash="2ce483f3">2012.amta-keynotes.4</url>
+    </paper>
+    <paper id="5">
+      <title>On Marketing and Poetry</title>
+      <author><first>Caitilin</first><last>Walsh</last></author>
+      <url hash="03703a7c">2012.amta-keynotes.5</url>
+    </paper>
+    <paper id="6">
```

Comment:
This should be the first one in the oral session.

Reply:
Hi David—you marked this as resolved, but didn't move it? According to [this page](https://aclanthology.org/www.mt-archive.info/10/AMTA-2012-TOC.htm), this paper (#6) should be #1 in the "papers" volume, but you have it in the "keynotes" volume. This will require renumbering all papers there.